src/main/java/com/projetogerenciamentocurso/gerenciamentocurso/models/Aluno.java

```diff
@@ -22,40 +23,19 @@
 @NoArgsConstructor
 @EqualsAndHashCode(callSuper=false)
 public class Aluno extends Pessoa implements Serializable{
-	
-	private static final long serialVersionUID = 1L;
+
+	private static final long serialVersionUID = -6440400741036979763L;
 	private Integer matricula;
 	@NotNull
 	private String formaIngresso;
 	@ManyToMany(fetch = FetchType.LAZY,cascade = CascadeType.ALL)
 	private Set<Turma> turma = new HashSet<>();
 	
 	
-	public Aluno(Integer idPessoa, String nome, String cpf, String email, Integer matricula, String formaIngresso,
+	public Aluno(Integer idPessoa, String nome, String cpf, String email, String formaIngresso,
 			Set<Turma> turma) {

```

Comment:
Retirar esse parâmetro, pois ele não está sendo utilizado
```suggestion
) {
```

Reply:
feito

Reply:
Correção não foi aplicada

Reply:
feito